reporter-web-app/src/components/SummaryView.js

```diff
@@ -218,7 +218,7 @@ class SummaryView extends React.Component {
                             </Item>
                             <Item
                                 dot={
-                                    (hasUnresolvedIssues || hasUnresolvedRuleViolations > 0)
+                                    (hasUnresolvedIssues || hasUnresolvedRuleViolations)
```

Comment:
It would be useful to add a link to the rule in the commit message to explain why this was reported by LGTM:
https://lgtm.com/rules/9980081/

Reply:
Done.